### PR TITLE
fix: postgres+redshift query default timeout

### PR DESCRIPTION
**Pull request by IrakliJani**
Closes: 

### Description:

test-frontend
test-backend
test-cli

log:

CleanShot 2024-12-20 at 02 04 13@2x

ui:

CleanShot 2024-12-20 at 02 03 50@2x

### Reviewer actions

- [ ] I have manually tested the changes in the preview environment
- [ ] I have reviewed the code
- [ ] I understand that "request changes" will block this PR from merging

### CODE REVIEW

File: packages/warehouses/package.json [hunk 1]
```diff
@@ -12,14 +12,14 @@
...
-        "pg": "^8.11.3",
+        "pg": "^8.13.1",
         "pg-cursor": "^2.10.0",
         "snowflake-sdk": "~1.14.0",
         "ssh2": "^1.14.0",
         "trino-client": "0.2.6"
     },
     "devDependencies": {
-        "@types/pg": "^8.10.7",
+        "@types/pg": "^8.11.10",
```
Comment (rephus) [inline packages/warehouses/package.json:22]: does this version need to match pg 8.13.1 ?

**Review by rephus — APPROVED** [review]
Code looks good

**Comment by rephus** [comment]
Unrelated ee2e test ? I've never seen this failing :thinking: 

image

**Comment by rephus** [comment]
SQL Runner (new) -- Should save a chart (failed) (attempt 2)

**Comment by rephus** [comment]
Testing recursive sql on render 

https://lightdash-pr-12980.onrender.com/projects/bc23a693-ff67-4984-bf05-f1c29cfdf082/tables/recursive

**Review by rephus — APPROVED** [review]
I couldn't get this error on render, but I'm happy to merge it and see if that solves the issue for the customer.

File: packages/warehouses/package.json (hunk 1 above)
Comment (IrakliJani) [reply packages/warehouses/package.json:22]: its on latest version

**Comment by IrakliJani** [comment]
I just rerun tests and they pass. merging the PR and going to add another chunk of work to introduce an UI for the timeout

**Pull request merged**